### PR TITLE
Add issue/pr templates

### PR DESCRIPTION
**Pull request by SeanZoR**

### CODE REVIEW

File: ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
```
Comment (workingrush) [inline ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:11]: Consider moving this section below "steps to reproduce", and write that everything after "steps to reproduce" is optional. This lowers the effort required to submit bug reports. 
If you are blasted with low quality bug reports, you can then change the template to require more details.

File: ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Issue tracker is **ONLY** used for reporting bugs. Support or new features should be discussed on [Lava's Discord channel](https://discord.gg/5VcqgwMmkA).
```
Comment (workingrush) [inline ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:1]: Consider that by asking users that are on your Github to go somewhere else, you could be losing out on their feedback.

Comment (SeanZoR) [reply ISSUE_TEMPLATE/ISSUE_TEMPLATE.md:1]: Thanks, it's an interim time until we fully accept all contributions.

File: ISSUE_TEMPLATE/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (SeanZoR) [reply ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]: Done 👍

**Pull request merged**